### PR TITLE
CRM-20006 Standard groups relationships in views

### PR DESCRIPTION
**Pull request by seamuslee001**
- Expose standard group contact Relationship and be able to filter on status
- Expose standard group Relationship for active and non hidden groups, and be able to filter on type and name
- fix bug preventing to use the field 'group status'
- add filter on group name (for cross-sites export)
- Expose group custom data to group relationship.

@jitendrapurohit this is a re-submit of https://github.com/civicrm/civicrm-drupal/pull/427 are you able to re-test?

---

 * [CRM-20006: Drupal Views : relationships for standard groups](https://issues.civicrm.org/jira/browse/CRM-20006)

### CODE REVIEW

**Comment by jitendrapurohit** [comment]
Thanks @seamuslee001. I can test this later this week.

File: modules/views/components/civicrm.core.inc [hunk 1]
```diff
@@ -2816,6 +2863,23 @@ function _civicrm_core_data(&$data, $enabled) {
...
+      'click sortable' => TRUE,
+    ),
+    'argument' => array(
+      'handler' => 'views_handler_argument',
+    ),
+    'sort' => array(
+      'handler' => 'views_handler_sort',
+    ),
+  );
+
```
Comment (jitendrapurohit) [inline modules/views/components/civicrm.core.inc:2983]: The new title is not included in view filters. Was this intentional? Though it contains the deprecated title.

**Comment by jackrabbithanna** [comment]
@seamuslee001 I think if you address jitendrapurohit's question then after that this should be merged.

File: modules/views/components/civicrm.core.inc (hunk 1 above)
Comment (seamuslee001) [reply modules/views/components/civicrm.core.inc:2983]: Not sure why but have updated it now @jackrabbithanna

**Comment by seamuslee001** [comment]
@jitendrapurohit are you able to re-test this, looks like it was originally one of yours i rebased to get sorted would be good to get this merged

**Review by jitendrapurohit — APPROVED** [review]
Have tested this and confirm that the standard groups are correctly exposed in fields, filter and relationship section of the drupal view. +1 to get this included in core.

Thanks @seamuslee001

**Comment by eileenmcnaughton** [comment]
Merging per review by @jitendrapurohit & comments from @jackrabbithanna indicating this is an adequate review

**Pull request merged**